key/soft_key.go

```diff
@@ -279,15 +278,7 @@ func (m *SoftKey) PrivKeyHex() string {
 
 // Saves the private key to disk with hex encoding.
 func (m *SoftKey) Save(p string) error {
-	return os.WriteFile(p, []byte(m.PrivKeyHex()), constants.UserOnlyWriteReadPerms)
-}
-
-func (m *SoftKey) P(networkHRP string) (string, error) {
-	return address.Format("P", networkHRP, m.privKey.PublicKey().Address().Bytes())
-}
-
-func (m *SoftKey) X(networkHRP string) (string, error) {
-	return address.Format("X", networkHRP, m.privKey.PublicKey().Address().Bytes())
+	return os.WriteFile(p, []byte(m.PrivKeyHex()), constants.WriteReadUserOnlyPerms)
 }
 
 func (m *SoftKey) Spends(outputs []*avax.UTXO, opts ...OpOption) (
```

Comment:
This file also contains `Match()` function that is not obvious. Can we add description?

Reply:
https://github.com/ava-labs/avalanche-tooling-sdk-go/issues/173